### DISCUSSION TITLE
LXQt Themes

### DISCUSSION THREAD

**Discussion by edwardp3** [lxqt/lxqt#2254] [General]
Just want to give a shoutout to @stefonarch. Was looking for additional LXQt themes and the search results included his repository here. Installed ice, black-red and black-green. They all work perfectly with LXQt 0.16 on Mageia 8.

Excellent work. The wallpapers with them look great as well. Thank you!

**Comment by stefonarch** [comment]
Well, thank you very much :)

 Those are the only themes I didn't update and recheck (Valendas, Arch colors and Silver are in the official repo now) - I suspect them being  full of issues   with dark palettes an else but 0.16 doesn't have them.

**Reply by edwardp3** [reply]
I wouldn't mind trying the same themes using LXQt 1.x, but I'm likely going to have to wait until Mageia 9 is released.

**Reply by edwardp3** [reply]
After I posted the above, I had a thought. I loaded in a live image of GeckoLinux LXQt which provides openSUSE Tumbleweed, the image included LXQt 1.0.0.

I went to your repository and retrieved/extracted the same three themes. Happy to report that in 1.0.0, they all displayed perfectly, no issues. :)

**Reply by stefonarch** [reply]
Themes have no versioning :)

**Reply by edwardp3** [reply]
All of the themes installed, including from the LXQt repo here, display fine with 0.16.0. 

KDE Plasma and System are duplicated in the Themes list, as it's pulling them in from both the system and my .local/share/lxqt/themes folders, so I added (New) to the KDE-Plasma and System folder names in my /themes folder. I recall the original System theme was missing the LXQt bird menu icon at the time, but it's present in the new theme.

The more I use and customize LXQt, the more I like it. :)

**Reply by dglent** [reply]
> I wouldn't mind trying the same themes using LXQt 1.x, but I'm likely going to have to wait until Mageia 9 is released.

Awaiting Mageia 9, you can install LXQt 1.1.0 for Mageia 8 from 3rd part repository here: http://www.mageia-gr.org/rpm/8/x86_64/

**Reply by edwardp3** [reply]
@stefonarch I saved Ice, Black Red and Black Green to a thumb drive and copied them into my Debian install. They all work perfectly with LXQt 1.2.0. I also like the way that themes placed in `/.local/share/lxqt/themes` now have **(User Theme)** next to the name in the listing.

**Reply by stefonarch** [reply]
Uh, a part from the fact that I don't like them  much anymore - I'm sure they are full of small issues (for example with a dark palette). 
 Here's a list from themes we heard about:
https://github.com/lxqt/lxqt/wiki/Theming#community-created-themes

**Reply by edwardp3** [reply]
From the information on the above page, I installed LXQt-Breeze (which installed two themes) and Lush.

**Reply by AzumaHazuki** [reply]
Mahe's Breeze themes are amazing, huh? They may be the most comprehensive and complete themes so far extant, and they look almost scarily accurate.

I have <a href=https://github.com/AzumaHazuki/lxqt-themes-mint-y>some WIP</a> <a href=https://github.com/AzumaHazuki/lxqt-themes-materia>theme sets</a> <a href=https://github.com/AzumaHazuki/lxqt-themes-arc>you</a> <a href=https://github.com/AzumaHazuki/lxqt-themes-adwaita>might like</a> as well :) These are subject to rewriting when I find the time (as no one seems to know all the widgets we can theme, but we're <a href=https://github.com/lxqt/lxqt/wiki/Creating-LXQt-Themes>working on that</a> too...). 

Eventually I'd like to get the Adwaita themes at least into the base set given how ubiquitous the look is, and maybe have packages available for the others...who knows, maybe once perfected that Mint-Y set will lead to an LXQt spin of Linux Mint =P

**Reply by edwardp3** [reply]
The light Breeze theme included with LXQt-Breeze, looks exactly like the KDE-Plasma theme included with LXQt. I noticed with KDE-Plasma, the keyboard indicator applet letters (C N S) are shifted down a little, inside the panel, with the blue line indicators above them, but the light Breeze theme has the same letters more centered in the panel with the same indicators. I don't know if in either instance, that's by design.

I had to stop using a compositor with LXQt, because my web browser's UI kept jumping on screen when its window was maximized. I have no idea why _that_ occurs, but while using openbox for the window manager, the UI doesn't jump.

**Reply by edwardp3** [reply]
@AzumaHazuki I just tried to install your Adwaita and Materia themes, but after extraction to `/.local/share/lxqt/themes`, neither were listed in the LXQt Appearance Configuration window.

**Reply by stefonarch** [reply]
Probably they finished in a subfolder. Copying /themes/Adwaita* to _~/.local/share/lxqt/themes_ works as expected.

**Reply by edwardp3** [reply]
@stefonarch There were two sets of folders with the theme names, once the .zip was extracted. The wrong folders were initially copied into ~/.local/share/lxqt/themes. Works now. Thanks.

**Reply by AzumaHazuki** [reply]
> @AzumaHazuki I just tried to install your Adwaita and Materia themes, but after extraction to `/.local/share/lxqt/themes`, neither were listed in the LXQt Appearance Configuration window.

Glad to hear they're working now :) I actually updated them a bit last night so they may look a little different now. Thank you for (inadvertently!) beta-testing these; I _think_ they're ready to be PR'd but would love any feedback you can offer!

**Reply by edwardp3** [reply]
@AzumaHazuki I looked at all of your repos and the only one that had an update showing today was Adwaita, so I downloaded that .zip file. _Once a repo here is updated, I don't know how long it takes for any updates to be available._ 

I renamed the original Adwaita themes to include `Beta`, so as not to delete/overwrite them once the .zip was extracted.

Both Adwaita (light) and Adwaita Dark (new versions) look great, using Openbox for the Window Manager. The current Materia themes look great as well.

**Comment by edwardp3** [comment]
@AzumaHazuki I believe I just found a bug in both of the Materia themes, relating to the Keyboard State Indicator applet. If the `C N S` letters should be bolded when the respective lock key is active on the keyboard, all three letters are bolded in both themes. Pressing any of the three lock keys on the keyboard does not change the behavior.

The `US` in the screenshots below, is part of the same applet and is included for comparison.

![materia](https://user-images.githubusercontent.com/73129450/211639210-d1fdd46d-e801-4408-b001-d73b5ed930c5.png)

![materia-dark](https://user-images.githubusercontent.com/73129450/211639215-3c8321e0-5cf0-4297-a60e-79d196a2530a.png)

**Reply by AzumaHazuki** [reply]
Thank you for testing these out! I'm actually in the middle of refactoring the Adwaita set and will use what I learn to rewrite the others in the same/similar format, as well as likely update Leech and Clearlooks. Some features that aren't yet widely-known are becoming active, such as styling the Leave dialog and the as-yet-unintegrated WingMenu extension.

It seems like the bug isn't the bolding or lack thereof, but that the color/shade is supposed to change and doesn't. Still debating the best way to indicate on/off for these.

**Reply by AzumaHazuki** [reply]
Okay, I've made a PR and also uploaded the (hopefully...) final versions to https://github.com/AzumaHazuki/lxqt-themes-adwaita for safekeeping. _Think_ i got everything this time! It's even got styling for the logout dialog and the Wing Menu.

**Comment by M4he** [comment]
In case you haven't found this yet, I created 2 themes matching the latest rendition of the Breeze visual style of KDE 5.24 and later: https://gitlab.com/M4he/lxqt-theme-breeze

**Reply by stefonarch** [reply]
We have :)
https://github.com/lxqt/lxqt/wiki/Theming#community-created-themes
 But people read google and not wikis or manuals...

**Reply by edwardp3** [reply]
@M4he I have those Breeze themes installed. Excellent work.  :)
